packages/manager/apps/web/client/app/hosting/ftp/hosting-ftp-constants.js

```diff
@@ -0,0 +1,15 @@
+export const SSH_STATE = {
+  ENABLED: 'active',
+  SFTP_ONLY: 'sftponly',
+  DISABLED: 'none',
```

Comment:
Why not naming the constants to match their value?

Reply:
The idea was to match text displayed to the user, but thinking about it it makes more sense to name it as you propose.